stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/theme/ChatTheme.kt

```diff
@@ -84,7 +85,7 @@ public fun ChatTheme(
     shapes: StreamShapes = StreamShapes.defaultShapes(),
     attachmentFactories: List<AttachmentFactory> = StreamAttachmentFactories.defaultFactories(),
     attachmentPreviewHandlers: List<AttachmentPreviewHandler> = AttachmentPreviewHandler.defaultAttachmentHandlers(LocalContext.current),
-    reactionTypes: Map<String, Int> = DefaultReactionTypes.defaultReactionTypes(),
+    reactionIconFactory: ReactionIconFactory = ReactionIconFactory.defaultFactory(),
```

Comment:
What do you think about the naming? Basically mostly responsible for icon creation.